addons/website/views/snippets/s_cards_soft.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_cards_soft" name="Cards Soft">
+    <section class="s_cards_soft o_cc o_cc1 pt32 pb32 o_colored_level" data-vxml="001" data-vcss="001">
+        <div class="container">
+            <h2 style="text-align: center;">Your Journey Begins Here</h2>
+            <p class="lead" style="text-align: center;">We make every moment count with solutions designed just for you.</p>
+            <div class="row d-flex align-items-stretch gx-3">
+                <div data-name="Card" class="col-lg-4 pt16 pb16">
+                    <div class="s_card o_card_img_top card rounded p-4 my-0 o_cc o_cc2" data-vxml="001" data-name="Card" data-snippet="s_card" style="border-width: 0 !important; gap: 1rem;">
+                        <div class="card-body p-0">
+                            <h5 class="card-title">Innovative Ideas</h5>
+                            <p class="card-text">Our creativity is at the forefront of everything we do, delivering innovative solutions that make your project stand out while maintaining a balance between originality and functionality.</p>
+                        </div>
+                        <div class="o_card_img_wrapper ratio ratio-4x3">
+                            <img class="o_card_img card-img-top rounded" src="/web/image/website.s_three_columns_default_image_1" alt=""/>
+                        </div>
+                    </div>
```

Comment:
We can get rid of a lot of classes that are not editor compatible by doing something like this:


```suggestion
                    <div class="s_card rounded o_cc o_cc2" data-vxml="001" data-name="Card" data-snippet="s_card" style="border-width: 0 !important;">
                        <div class="card-body">
                            <h5 class="card-title">Innovative Ideas</h5>
                            <p class="card-text">Our creativity is at the forefront of everything we do, delivering innovative solutions that make your project stand out while maintaining a balance between originality and functionality.</p>
                            <div class="o_card_img_wrapper ratio ratio-4x3">
                                <img class="o_card_img rounded" src="/web/image/website.s_three_columns_default_image_1" alt=""/>
                            </div>
                        </div>
                    </div>
```